src/RadworksGovernor.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {Governor} from "@openzeppelin/contracts/governance/Governor.sol";
+import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {
+  ERC20VotesComp,
+  GovernorVotesComp
+} from "@openzeppelin/contracts/governance/extensions/GovernorVotesComp.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {ICompoundTimelock} from
+  "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {GovernorSettings} from "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
+
+import {IRadworksTimelock} from "src/interfaces/IRadworksTimelock.sol";
+import {GovernorTimelockCompound} from "src/lib/GovernorTimelockCompound.sol";
+import {GovernorCompatibilityBravo} from
+  "@openzeppelin/contracts/governance/compatibility/GovernorCompatibilityBravo.sol";
+
+/// @notice The upgraded Radworks Governor: Bravo compatible and built with OpenZeppelin.
+contract RadworksGovernor is
+  Governor,
+  GovernorVotesComp,
+  GovernorTimelockCompound,
+  GovernorSettings
+{
+  struct ProposalVote {
+    uint256 againstVotes;
+    uint256 forVotes;
+    uint256 abstainVotes;
+  }
+
+  /// @notice The address of the RAD token on Ethereum mainnet from which this Governor derives
+  /// delegated voting weight.
+  ERC20VotesComp private constant RAD_TOKEN =
+    ERC20VotesComp(0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3);
+
+  /// @notice The address of the existing Radworks DAO Timelock on Ethereum mainnet through
+  /// which
+  /// this Governor executes transactions.
+  IRadworksTimelock private constant TIMELOCK =
+    IRadworksTimelock(payable(0x8dA8f82d2BbDd896822de723F55D6EdF416130ba));
+
+  /// @notice Human readable name of this Governor.
+  string private constant GOVERNOR_NAME = "Radworks Governor Bravo";
+
+  /// @notice The number of RAD (in "wei") that must participate in a vote for it to meet quorum
+  /// threshold.
+  ///
+  /// TODO: placeholder
+  uint256 private constant QUORUM = 100_000e18; // 100,000 RAD
+
+  /// @param _initialVotingDelay The deployment value for the voting delay this Governor will
```

Comment:
nit: i see these are documented like this elsewhere, so feel free to ignore -- was thinking this might be more concise

```suggestion
  /// @param _initialVotingDelay The initial voting delay this Governor will
```

Reply:
Done 4c8b4aa5079ff2ca8a0124eeab3624cc5cfce0db